### PR TITLE
RELATED: RAIL-3439 fix versions table

### PR DESCRIPTION
**Pull request by no23reason**
Only the last of the previous major should be End of development
others need to be End of support.

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/736/ : SUCCESS in 1m 18s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/469/ : SUCCESS in 11s

File: website/pages/en/versions.js [hunk 1]
```diff
@@ -14,19 +14,20 @@ const semverSatisfies = require("semver").satisfies;
 const semverMinimal = require("semver").minSatisfying;
 const semverMaximal = require("semver").maxSatisfying;
 const semverParse = require("semver").parse;
+const semverMaxSatisfying = require("semver").maxSatisfying;
```
Comment (kandl) [inline website/pages/en/versions.js:17]: Duplicate of semverMaximal

Comment (no23reason) [reply website/pages/en/versions.js]: Good catch, fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/737/ : SUCCESS in 54s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/470/ : SUCCESS in 6s

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR420

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/738/ : SUCCESS in 53s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/117/ : SUCCESS in 41s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/19/ : SUCCESS in 42s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/208/ : SUCCESS in 16s

**Pull request merged**